src/main/java/core/basesyntax/model/RightTriangle.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure implements Drawable {
```

Comment:
The `RightTriangle` class should implement methods to calculate the area and set a color, as these are likely required by the task. Ensure that the class has fields for the dimensions necessary to calculate the area and a method to set the color.